packages/manager/src/features/DataStream/Streams/StreamCreate/StreamCreate.tsx

```diff
@@ -1,18 +1,30 @@
 import { Stack } from '@linode/ui';
 import Grid from '@mui/material/Grid';
 import * as React from 'react';
+import { FormProvider, useForm } from 'react-hook-form';
 
 import { DocumentTitleSegment } from 'src/components/DocumentTitle';
 import { LandingHeader } from 'src/components/LandingHeader';
-import { StreamCreateCheckoutBar } from 'src/features/DataStream/Streams/StreamCreate/StreamCreateCheckoutBar';
-import { StreamCreateDataSet } from 'src/features/DataStream/Streams/StreamCreate/StreamCreateDataSet';
-import { StreamCreateDelivery } from 'src/features/DataStream/Streams/StreamCreate/StreamCreateDelivery';
-import { StreamCreateGeneralInfo } from 'src/features/DataStream/Streams/StreamCreate/StreamCreateGeneralInfo';
 
 import { useStyles } from './StreamCreate.styles';
+import { StreamCreateCheckoutBar } from './StreamCreateCheckoutBar';
+import { StreamCreateDataSet } from './StreamCreateDataSet';
+import { StreamCreateDelivery } from './StreamCreateDelivery';
+import { StreamCreateGeneralInfo } from './StreamCreateGeneralInfo';
+import { type CreateStreamForm, EventType, StreamType } from './types';
 
 export const StreamCreate = () => {
   const { classes } = useStyles();
+  const form = useForm<CreateStreamForm>({
+    defaultValues: {
+      type: StreamType.AuditLogs,
+      [EventType.Authorization]: false,
+      [EventType.Authentication]: false,
+      [EventType.Configuration]: false,
```

Comment:
nit: including a default value for the label since its type is string in `CreateStreamForm` (not sure if not having this would cause type issues down the line when trying to submit)

```suggestion
      [EventType.Configuration]: false,
      label: '',
```